src/managers/GuildChannelManager.js

```diff
@@ -177,6 +178,27 @@ class GuildChannelManager extends CachedManager {
     for (const channel of data) channels.set(channel.id, this.client.channels._add(channel, this.guild, { cache }));
     return channels;
   }
+
+  /**
+   * Obtains all active thread channels in the guild from Discord
+   * @param {boolean} [cache=true] Whether to cache the fetched data
+   * @param {boolean} [returnRaw=false] Whether to return the raw data
+   * @returns {Promise<FetchedThreads>|Promise<APIActiveThreadsList>}
+   * @example
+   * // Fetch all threads from the guild
+   * message.guild.channels.fetchActiveThreads()
+   *   .then(fetched => console.log(`There are ${fetched.threads.size} threads.`))
+   *   .catch(console.error);
+   */
+  async fetchActiveThreads(cache = true, returnRaw = false) {
+    const raw = await this.client.api.guilds(this.guild.id).threads.active.get();
+    return returnRaw ? raw : ThreadManager._mapThreads(raw, this.client, { guild: this.guild, cache });
```

Comment:
This (`returnRaw`) feels a bit out of place, we don't do this anywhere else, do we?

Reply:
what would be the use case for returning the raw threads anyway?

Reply:
its for internals, see
https://github.com/discordjs/discord.js/blob/150a295224ea0de5ea541d624addc8489e00b44a/src/managers/ThreadManager.js#L232

Reply:
I don't think that should be part of the public API then. :eyes:
Maybe move that to a private method or duplicate that api call into the ThreadManager?

Reply:
I was thinking about duplicating the call, especially since its a one-liner. Is that better?

Reply:
I think so.